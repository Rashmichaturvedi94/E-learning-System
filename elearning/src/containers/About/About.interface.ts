export interface AboutProps {
  // TODO: add required props below
  myProp?: boolean;
}
