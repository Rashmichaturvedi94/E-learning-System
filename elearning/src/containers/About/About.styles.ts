import styled from 'styled-components/native';
import { Text } from 'react-native';
import { AboutProps } from './About.interface';

export const Title = styled(Text)`
  text-align: center;
  font-size: 30px;
  font-weight: bold;
  margin-top: 15px;
  margin-bottom: 20px;
`;

export const Version = styled(Text)`
  color: gray;
  font-size: 20px;
  font-weight: bold;
  margin-left: 15px;
  margin-top: 15px;
  margin-bottom: 10px;
`;

export const Info = styled(Text)`
  color: gray;
  font-size: 20px;
  font-weight: bold;
  margin-left: 15px;
  margin-top: 10px;
  margin-bottom: 20px;
`;
