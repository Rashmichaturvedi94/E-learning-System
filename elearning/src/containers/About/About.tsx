import React, { FC } from 'react';
import { View } from 'react-native';
import { Title, Version, Info } from './About.styles';
import { AboutProps } from './About.interface';

export const About = () => {
  return (
    <View>
      <Title>About Us</Title>
      <Version>Version 1.0</Version>
      <Info>
        This application provides the platform to learner and it is available
        around the clock over the internet using mobile application.
        Students/learner will be able to access the materials available in the
        system uploaded by authors/educational institutions. The System will
        provide the post-secondary education which will be available in english
        language.
      </Info>
    </View>
  )
};
