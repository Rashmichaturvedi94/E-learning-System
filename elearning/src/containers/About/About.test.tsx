import React from 'react';
import { render } from '@testing-library/react-native';
import { About } from './About';

describe('containers/About', () => {
  it('should render', () => {
    const component = render(<About />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
