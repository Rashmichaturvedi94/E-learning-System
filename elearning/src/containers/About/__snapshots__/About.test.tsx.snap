// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`containers/About should render 1`] = `
<View>
  <Text
    style={
      Array [
        Object {
          "fontSize": 30,
          "fontWeight": "bold",
          "marginBottom": 20,
          "marginTop": 15,
          "textAlign": "center",
        },
      ]
    }
  >
    About Us
  </Text>
  <Text
    style={
      Array [
        Object {
          "color": "gray",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 10,
          "marginLeft": 15,
          "marginTop": 15,
        },
      ]
    }
  >
    Version 1.0
  </Text>
  <Text
    style={
      Array [
        Object {
          "color": "gray",
          "fontSize": 20,
          "fontWeight": "bold",
          "marginBottom": 20,
          "marginLeft": 15,
          "marginTop": 10,
        },
      ]
    }
  >
    This application provides the platform to learner and it is available around the clock over the internet using mobile application. Students/learner will be able to access the materials available in the system uploaded by authors/educational institutions. The System will provide the post-secondary education which will be available in english language.
  </Text>
</View>
`;
